uint8 PLAY=0
uint8 PAUSE=1
uint8 NEXT=2
uint8 PREVIOUS=3
uint8 player_action
---
string song_name
float32 audio_priority
